workflows-scripts/circulation/check-in-check-out/scripts/checkin-checkout-db-restore.sql

```diff
@@ -0,0 +1,20 @@
+/*

```

Comment:
Is this script run **manually** before and after test?

Reply:
Yes, it's supposed to be run manually after each test.